src/main/java/org/mule/extension/http/api/error/HttpError.java

```diff
@@ -132,7 +159,11 @@
       try {
         error = HttpError.valueOf(status.name());
```

Comment:
this is still throwing an exception for the new scenario...

Reply:
This piece of code is the same as before my change.  If `HttpError#valueOf` throws an exception, it's caught silently ("// Do nothing"), and `error` remains null.  In this case, the fallback will search for a category-related error.

Reply:
this is a good opportunity to improve it :)

Reply:
done